tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -4817,6 +4898,33 @@ def _validate_fit_data(
                 )
         return train_data, tuning_data, test_data, unlabeled_data
 
+    def _initialize_learning_curve_params(self, learning_curves):
+        if learning_curves is None or learning_curves == False:
+            return {}
+        elif type(learning_curves) != dict and type(learning_curves) != bool:
+            raise ValueError("VALUE ERROR: learning curves parameter must be a boolean or dict!")
+
+        use_error = True
+        metrics = DEFAULT_LEARNING_CURVE_METRICS[self.problem_type]
+
+        if type(learning_curves) == dict:
+            if "metrics" in learning_curves:
+                if type(learning_curves["metrics"]) == str:
+                    metrics = [learning_curves["metrics"]]
```

Comment:
Is there a possibility of having multiple metrics?
rmse, mse ?
if yes this it becomes a list of lists ?